invenio_search/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2015, 2016, 2017 CERN.
```

Comment:
you only need to add the year you made the change :P

Reply:
Ahh, I didn't know that! I thought we always add all the missing years. Thanks!

Reply:
it's just a detail :)

Reply:
Anyway, this file was also touched in `2016` :) (23acaa61301276a325e90)